fixtures/vulnerable-demo/.env.leaky

```
@@ -0,0 +1,3 @@
+OPENAI_API_KEY=sk-proj-THISISNOTAREALKEY012345678901234567890
+DATABASE_URL=postgres://runtime_user:runtime_password@db.internal:5432/app
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DATABASE_URL key should go before the OPENAI_API_KEY key

(UnorderedKey)